kalite/distributed/static/js/distributed/topics/views.js

```diff
@@ -104,28 +115,33 @@ window.SidebarView = BaseView.extend({
     },
 
     resize_sidebar: function() {
-        if ($(window).width() < 1260)
-            this.resize_for_narrow();
+        if (this.state_model.get("open")) {
+            if ($(window).width() < 1260)
+                this.resize_for_narrow();
```

Comment:
Ooh I didn't know we were doing that. I always add them when needed and ignore when I can. What linter? Is it something on Travis?

> On Mar 9, 2015, at 4:01 PM, Michael Gallaspy <notifications@github.com> wrote:
> 
> In kalite/distributed/static/js/distributed/topics/views.js:
> 
> > @@ -104,28 +115,33 @@ window.SidebarView = BaseView.extend({
> >      },
> >  
> >      resize_sidebar: function() {
> > -        if ($(window).width() < 1260)
> > -            this.resize_for_narrow();
> > +        if (this.state_model.get("open")) {
> > +            if ($(window).width() < 1260)
> > +                this.resize_for_narrow();
> Noooo! Use the curly brackets! I don't know why this passed the linter. (If you truly must omit curly brackets, then at least don't put the statement to be executed on a new line... it suggests that more lines could be added to this if block, but really only the first statement will be conditionally executed and all subsequent ones will be unconditionally executed).
> 
> —
> Reply to this email directly or view it on GitHub.
>